README.md

```diff
@@ -183,18 +183,46 @@ Once the browser is open to your Lookout page (see above steps), you can:
 ## Using the command line to control your CGM
 The commands below can be entered on the rig command line to control the CGM. Regardless of which command is entered, after executing the command the command will enter a status loop indefinately printing the CGM status at each glucose read event. Enter `Ctrl-C` to exit the command.
 ```
-  lookout cal <sgv>                  # Calibrate the transmitter with provided glucose meter reading
-  lookout id <id>                    # Set transmitter ID
-  lookout meterid <meterid>          # Set transmitter ID
-  lookout start [sensor_serial]      # Start sensor session; sensor serial required for G6
-  lookout back-start [sensor_serial] # Start sensor session back dated by 2 hours; sensor serial required for G6
+  lookout cal <sgv>                  # Calibrate the transmitter with glucose meter reading. Example: `lookout cal 100`
+  lookout id <id>                    # Set transmitter ID. Example: `lookout id 9515`
+  lookout meterid <meterid>          # Set fakemeter ID. Example: `lookout meterid 123456`
+  lookout start [sensor_serial]      # Start sensor session; sensor serial required for G6. Example: `lookout start [9515]`
+  lookout back-start [sensor_serial] # Start sensor session back dated by 2 hours; sensor serial required for G6. 
+                                       Example: `lookout back-start [9515]`
   lookout stop                       # Stop sensor session
   lookout reset                      # Reset transmitter
   lookout status                     # Show status  [default]
 ```
 
 Use `-m` option for mmol instead of mg/dL. For example, `lookout -m cal 4.1` will calibrate with 4.1 mmol.
 
+## Replacing a Sensor, using the command line
+This assumes that a sensor session is active, and that you are using the same transmitter for both sessions. The goal is to minimize looping-downtime during the sensor change. This should work with a Dexcom G6 sensor.
+
+All commands as shown are entered from the Lookout directory level. To get into the Lookout directory, if you are watching a log etc., - start with Enter `Ctrl-C`, then enter `cd` to get to root level, then enter `cd Lookout`to get into the Lookout Directory. Note that "Lookout" is capitalized.
+
+### Pre-soak the sensor
+1. Insert new sensor some time before old sensor is ready to come out (2+ hours). 
+Note the sensor serial number, you'll want it later to start, to restart the sensor, and for troubleshooting. (take a photo ...)
```

Comment:
Reword to `If using a G6, note the sensor serial number.  You'll want it later to start, to restart the sensor, and for troubleshooting. (take a photo...)`